src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeUnknownRiskCellConfigurator.swift

```diff
@@ -88,4 +88,20 @@ final class HomeUnknownRiskCellConfigurator: HomeRiskLevelCellConfigurator {
 
 		setupAccessibility(cell)
 	}
+
+	// MARK: Hashable
+
+	override func hash(into hasher: inout Swift.Hasher) {
+		super.hash(into: &hasher)
+		hasher.combine(detectionInterval)
+	}
+
+	static func == (lhs: HomeUnknownRiskCellConfigurator, rhs: HomeUnknownRiskCellConfigurator) -> Bool {
+		lhs.isLoading == rhs.isLoading &&
```

Comment:
same as above, so many additional attributes in == method, a comment would suffice

Reply:
see my answer for `HomeHighRiskCellConfigurator`